package.json

```diff
@@ -11,50 +11,50 @@
     "vanilla/*"
   ],
   "scripts": {
-    "bootstrap": "lerna bootstrap",
-    "lerna": "lerna",
-    "clean": "lerna run --parallel clean",
-    "build": "lerna run --parallel build",
-    "lint": "lerna run --parallel lint:es",
-    "lint:fix": "lerna run --parallel lint:es:fix",
-    "tsc": "lerna run --parallel tsc",
-    "test": "lerna run --parallel test",
+    "clean": "npm run clean --workspaces --if-present",
+    "clean:all": "rm -rf node_modules && npm run clean:all --workspaces --if-present",
+    "build": "npm run build --workspaces --if-present",
+    "lint": "npm run lint --workspaces --if-present",
+    "lint:fix": "npm run lint:fix --workspaces --if-present",
+    "tsc": "npm run tsc --workspaces --if-present",
```

Comment:
Should be run as part of the `test` command in each workspace then remove.

Reply:
Again, convenience. I run these commands over and over again when upgrading packages.

I have another round of package upgrades coming up. There are whole bunch of dependabot issues and warnings to resolve.